_data/offshore_regions.csv

```diff
@@ -1,6 +1,6 @@
 name,id,viewbox
-Alaska,alaska,75 415 202 173
-Atlantic,atlantic,730 125 230 440
-Gulf,gulf,470 440 295 135
-Pacific,pacific,20 60 161 341
-Pacific alaska,pacific alaska,20 60 180 530
+Alaska (offshore),alaska,75 415 202 173
+Atlantic (offshore),atlantic,730 125 230 440
+Gulf (offshore),gulf,470 440 295 135
```

Comment:
Yep! Anything that distinguishes it from other things. So -- still Offshore Alaska? And then what you suggest for the rest?